### PR TITLE
feat: ensure consistent hash for operations

### PR DESCRIPTION
**Pull request by StarpTech**
## Checklist

- [x] I have discussed my proposed changes in an issue and have received approval to proceed.
- [ ] I have followed the coding standards of the project.
- [x] Tests or benchmarks have been added or updated.
- [ ] Documentation has been updated on [https://github.com/wundergraph/cosmo-docs](https://github.com/wundergraph/cosmo-docs).
- [x] I have read the [Contributors Guide](https://github.com/wundergraph/cosmo/blob/main/CONTRIBUTING.md).

FIXES ENG-6013

### CODE REVIEW

**Review by jensneuse — APPROVED** [review]

File: router-tests/normalization_test.go [hunk 1]
```diff
@@ -11,7 +16,7 @@ import (
 )
 
 func TestNormalization(t *testing.T) {
-	t.Parallel()
+	//t.Parallel()
```
Comment (Noroth) [inline router-tests/normalization_test.go:19]: Was this intended?

File: router-tests/normalization_test.go [hunk 2]
```diff
@@ -1,6 +1,11 @@
 package integration
 
 import (
+	"github.com/wundergraph/cosmo/router/pkg/otel"
```
Comment (Noroth) [inline router-tests/normalization_test.go:4]: Sort imports

**Review by Noroth — COMMENTED** [review]
Minor nits

File: router-tests/normalization_test.go (hunk 1 above)
Comment (JivusAyrus) [reply router-tests/normalization_test.go:19]: Was testing if that was the issue. Reverting

Comment (JivusAyrus) [reply router-tests/normalization_test.go]: Done

**Review by Noroth — APPROVED** [review]

**Pull request merged**